guide/ops/troubleshooting/increase-entropy.md

```diff
@@ -4,24 +4,52 @@ title: Increase Entropy
 toc: /guide/toc.json
 ---
 
-If you are installing Apache Brooklyn on a virtual machine, you may find it useful to increase the 
-Linux kernel entropy to speed up the ssh connections to the managed entities. You can install and 
+### Checking entropy level
+
+A lack of entropy can cause random number generation to be extremely slow.
+This results in tasks like ssh to also be extremely slow.
+Check the available entropy on the Apache Brooklyn machine with.
+
+{% highlight bash %}
+cat /proc/sys/kernel/random/entropy_avail
+{% endhighlight %}
+
+It should be a value above 2000 
+
+If you are installing Apache Brooklyn on a virtual machine you will probably have little entropy you may need 
+to increase the Linux kernel entropy in order to speed up the ssh connections to the managed entities. You can install and 
 configure `rng-tools` or just use /dev/urandom`.
 
 
 ### Installing rng-tool
 
-if you are using a RHEL-based OS:
+If you are using a RHEL 6 based OS:
 
 {% highlight bash %}
+sudo -i
 yum -y -q install rng-tools
 echo "EXTRAOPTIONS=\"-r /dev/urandom\"" | cat >> /etc/sysconfig/rngd
 /etc/init.d/rngd start
 {% endhighlight %}
 
-if you are using a Debian-based OS:
+If you are using a RHEL 7 or a systemd based system:
+
+{% highlight bash %}
+sudo yum -y -q install rng-tools
+sudo vi /etc/systemd/system/multi-user.target.wants/rngd.service
+#
+# tell rng to use /dev/urandom
```

Comment:
This layout of commands is slightly confusing (i.e. the command above has done the vi, and then there's the same commented out vi below).

Change to:
```
sudo yum -y -q install rng-tools

# Configure rng to use /dev/urandom
# Change the "ExecStart" line to:
# ExecStart=/sbin/rngd -f -r /dev/urandom
sudo vi /etc/systemd/system/multi-user.target.wants/rngd.service

sudo systemctl daemon-reload
sudo systemctl start rngd
```